src/utils/visionAPI.py

```diff
@@ -0,0 +1,11 @@
+import os, io
+# from google.cloud import vision
+from google.cloud import vision_v1
+from google.cloud.vision_v1 import types
+
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = r'venv\ServiceAccountToken.json'
```

Comment:
You did you save the service account token file at venv folder? From tutorial I see that he save it besides of venv folder not inside it...

_If this step is correct, can you provide this file and keep it instead of ignore it?_

Reply:
I think service account client file should not be uploaded to the repo so everyone can reach, may you search for a better way to store that file.